app/api/__init__.py

```diff
@@ -28,12 +29,14 @@
 api.route(EventList, 'event_list', '/events')
 api.route(EventDetail, 'event_detail', '/events/<int:id>', '/tickets/<int:ticket_id>/event',
           '/microlocations/<int:microlocation_id>/event', '/social_links/<int:social_link_id>/event',
-          '/sponsors/<int:sponsor_id>/event', '/tracks/<int:track_id>/event', '/call_for_papers/<int:call_for_paper_id>/event')
+          '/sponsors/<int:sponsor_id>/event', '/tracks/<int:track_id>/event',
+          '/event_copyright/<int:copyright_id>/event', '/call_for_papers/<int:call_for_paper_id>/event')
```

Comment:
okay, afaik from what I asked @niranjan94 in https://github.com/fossasia/open-event-orga-server/pull/3718#discussion_r119982046, `/event_copyright/int:copyright_id` should'nt be there also.